DI-Policies.md

```diff
@@ -0,0 +1,67 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+      + Examples include, but are not limited to:
+         + Promotion of diversity-and-inclusion-minded vendors in my social feeds and blog
+         + Sharing love stories from underrepresented folx in my social feeds and blog
+
+   + Use my privilege, and speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website:
+      + Ensure representation of underrepresented groups in imagery across website, including blog
+      + Ensure the use of inclusive language (e.g. gender neutral terms) across website, including blog
+      + Ensure Diversity + Inclusion (D+I) Policies are up-to-date and accurate based on ongoing work and evolving best practices
+
+   + Monthly for social:
+      + Ensure representation of underrepresented groups in feeds, across all platforms (Instagram, Facebook, Twitter, Pinterest)
+      + Ensure the use of inclusive language extends to all social posts
+
++ Subcontractors, partners, + clients
```

Comment:
One final thing here - I think @b4mboo brought up a really good point about what you would do in the case of customers who display racist, homophobic or xenophobic behavior. I think adding in `clients` here and doing a shortened interview for them is an option, but I wonder if you may even want to add something into the clause of your contract that states that you can drop a client at any point if they display these behaviors, and then put an item in this doc stating that the clause exists, and you would drop them. Just something to think about!

Reply:
@bryanaknight ooooo, I like that idea! That's worth reaching out to my lawyer who drafted my contract I think. I'm going to inquire into this. Thank you!

Reply:
I actually double checked my contract and there's some vague language around this already. I just reached out to my lawyer to see what we can do about making it more explicit, because I really would love to have folks signing something that states they're committing to non-racist, -homophobic, etc behavior in order to have me work for them. Such a great idea; thank you @bryanaknight !

Reply:
This has been officially updated in my legal contract (my lawyer was so on top of it; bless that man). And I'll update the text here to reflect that. Thank you again @bryanaknight !